portfolio/src/main/webapp/script.js

```diff
@@ -94,3 +94,39 @@ function createTableElement(text) {
   trElement.appendChild(tdElementName);
   return trElement;
 }
+
+// Creates a map and adds it to the page
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 37.422, lng: -122.084}, zoom: 10});
```

Comment:
Can you use consts for the lat and the lng? Maybe COLUMBIA_UNIVERSITY_LAT.

Reply:
done